### PR TITLE
Add support for SwiftUI.Image

### PR DESCRIPTION
**Pull request by samuel-mellert**
I added an extension for SwiftUI.Image, comments and suggestions are welcome.

### CODE REVIEW

File: Sources/SFSafeSymbols/ImageExtension.swift [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+
+@available(iOS 13.0, *)
+public extension Image {
+
+    /// Creates a instance of `Image` with a `UIImage` object that has its rendering mode set to `.alwaysTemplate`.
+    ///
+    /// - systemSymbol: The `SFSymbol` describing this image.
+    @available(iOS 13.0, *)
+    init(systemSymbol: SFSymbol) {
+        self.init(uiImage: systemSymbol.toImage.withRenderingMode(.alwaysTemplate))
```
Comment (fredpi) [inline Sources/SFSafeSymbols/ImageExtension.swift:11]: Just wondering, whether we should use the `.alwaysTemplate` option everywhere... or make it accessible as an optional param? However, probably 100 % of the time, we want a template image, do you agree? If yes, would you mind implementing this everywhere? ;)

File: Sources/SFSafeSymbols/ImageExtension.swift [hunk 2]
```diff
@@ -0,0 +1,14 @@
+import SwiftUI
+
+@available(iOS 13.0, *)
+public extension Image {
+
+    /// Creates a instance of `Image` with a `UIImage` object that has its rendering mode set to `.alwaysTemplate`.
+    ///
+    /// - systemSymbol: The `SFSymbol` describing this image.
+    @available(iOS 13.0, *)
+    init(systemSymbol: SFSymbol) {
```
Comment (fredpi) [inline Sources/SFSafeSymbols/ImageExtension.swift:10]: Is there a specific reason there aren't initializers for `UIImage.Configuration` and `UITraitCollection`? If no, would you mind adding those too (Also to the README)?

**Review by fredpi — CHANGES_REQUESTED** [review]
Thanks for the PR 🚀 I made some suggestions, would you please have a look at them? ;)

Also, after having added this, we should restructure the tests a bit, but that should be in a separate PR as it would also affect the existing tests

File: Sources/SFSafeSymbols/ImageExtension.swift (hunk 1 above)
Comment (fredpi) [reply Sources/SFSafeSymbols/ImageExtension.swift:11]: Just had a look at the WWDC session about SF Symbols and they said all symbols are already rendered as `.alwaysTemplate`, so we won't need this ;)

Comment (samuel-mellert) [reply Sources/SFSafeSymbols/ImageExtension.swift:11]: When I tried in in Xcode with preview it didn't work, until I set the rendering mode to `.alwaysTemplate`. I'm watching the session right now. Let me get back to you afterwards.

**Comment by samuel-mellert** [comment]
Reworked and created new pull request.

**Pull request closed without merging**